clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3144,6 +3110,19 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
       }
     }
 
+    if (const auto *A = D->getAttr<SYCLIntelMaxGlobalWorkDimAttr>()) {
+      int64_t MaxGlobalWorkDim =
+          A->getValue()->getIntegerConstantExpr(Ctx)->getSExtValue();
```

Comment:
Shouldn't this be casting `getValue()` to a `ConstantExpr` (and if null, then we know the value is dependent)?